### PR TITLE
Person customizations (some outdated) to possibly merge

### PR DESCRIPTION
**Pull request by nathangibson**
@wsalesky Some of these changes (reversions to previous codebase) should probably **not** be adopted, but I put them here so you can confirm whether they should be discarded.

### CODE REVIEW

File: persons/facet-def.xml [hunk 1]
```diff
@@ -7,4 +7,106 @@
         <max-values show="5">40</max-values>
         <order-by direction="descending">count</order-by>
     </facet-definition>
+<!--
```
Comment (nathangibson) [inline persons/facet-def.xml:10]: If it doesn't hurt anything to keep these examples, they will be helpful as we develop further facets for persons.

File: persons/index.html [hunk 2]
```diff
@@ -1,11 +1,72 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
-    <div class="top-padding">
-        <div class="container">
-            <h1>Persons Module</h1>
-            <hr/>
-            <p>Use these pages to model your own subcollection pages.</p>
-            <h2>Example entries</h2>
-        </div>
+    <!--
```
Comment (nathangibson) [inline persons/index.html:3]: It looks like there's some d3 that should be kept in? But I'm not sure if the CSS classes are up to date.

File: persons/personsGraph.json [hunk 3]
```diff
@@ -0,0 +1,7656 @@
+{
```
Comment (nathangibson) [inline persons/personsGraph.json:1]: I assume you'll need this to generate graphs unless you have this data elsewhere.

File: persons/record.html [hunk 4]
```diff
@@ -1,29 +1,89 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
-    <div data-template="app:fix-links" class="record">
-        <div class="interior-content">
-            <div data-template="app:other-data-formats" data-template-formats="print,tei,rdf,text"/>
-            <div data-template="app:h1"/>
-            <div data-template="app:data-visualization" data-template-mode="force" data-template-locus="single"/>
-            <div data-template="app:display-nodes" data-template-paths="descendant::tei:text"/>
-            <!--
+    <div class="grey-dark">
```
Comment (nathangibson) [inline persons/record.html]: I couldn't tell if there's custom Usaybia content/styling here.

File: persons/search.html [hunk 5]
```diff
@@ -1,81 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
-    <div class="main-content-block">
```
Comment (nathangibson) [inline persons/search.html:3]: We haven't worked on the Usaybia search form much if any, so if this form is already generic, no need to revert. But I couldn't tell if there is some Syriaca-specific stuff here?

**Review by nathangibson — COMMENTED** [review]
@wsalesky Please check my comments. Incoming changes = reversions to the code from the srophe-exist-app repo.

File: persons/record.html (hunk 4 above)
Comment (wsalesky) [inline persons/record.html]: The persons record should revert to the version I had up originally, not the version from the old app. We do not use there is no longer a separate xquery module for persons, it was unnecessary. 

This is the essential code: 
```
<div data-template="app:other-data-formats" data-template-formats="print,tei,rdf,text"/>
            <div data-template="app:h1"/>
            <div data-template="app:data-visualization" data-template-mode="force" data-template-locus="single"/>
            <div data-template="app:display-nodes" data-template-paths="descendant::tei:text"/>
```

File: persons/record.html [hunk 6]
```diff
@@ -1,29 +1,89 @@
...
+        <div class="section interior-content white">
+            <!-- Page title and resource links: Includes h1, link icons, and item URI -->
+            <div data-template="person:h1"/>
+            <div data-template="app:other-data-formats" data-template-formats="print,tei,rdf,ttl,text"/>
             <div class="row">
-                <div class="col-md-7 col-lg-8">
```
Comment (wsalesky) [inline persons/record.html]: Bring back

File: persons/record.html [hunk 7]
```diff
@@ -1,29 +1,89 @@
...
-                    <div data-template="app:display-nodes" data-template-paths="descendant::tei:text"/>
```
Comment (wsalesky) [inline persons/record.html]: Bring back

File: persons/record.html [hunk 8]
```diff
@@ -1,29 +1,89 @@
...
+                    <div data-template="app:external-relationships" data-template-relationship-type="skos:broadMatch" data-template-collection="sbd"/>
+                    <div data-template="person:desc"/>
+                    <div data-template="person:authored-by"/>
+                    <div data-template="person:related-places"/>
+                    <div data-template="person:timeline" data-template-dates="personal"/>
+                    <div data-template="person:data"/>
+                    <!-- Citation -->
+                    <div data-template="person:sources"/>
                 </div>
-                <div class="col-md-5 col-lg-4 right-menu">
```
Comment (wsalesky) [inline persons/record.html]: Bring back

File: persons/record.html [hunk 9]
```diff
@@ -1,29 +1,89 @@
...
+                        </a>
+                    </div>
+                    <div data-template="person:relations"/>
+                    <div data-template="person:worldcat"/>
+                    <div data-template="person:link-icons-list"/>
                 </div>
             </div>
-            -->
-            <div>
-                <!-- Entry Citation information, pass TEI header to citation.xsl -->
```
Comment (wsalesky) [inline persons/record.html]: Bring back

**Review by wsalesky — CHANGES_REQUESTED** [review]
@nathangibson Most of these changes are fine. The persons/record.html should be reverted to the original version I had up, as the new version of the app does not have a separate persons module, it was unnecessary.

File: persons/facet-def.xml (hunk 1 above)
Comment (wsalesky) [reply persons/facet-def.xml:10]: This is fine.

File: persons/index.html (hunk 2 above)
Comment (wsalesky) [reply persons/index.html:3]: This is fine. We will need to update the data that is used by this graph at some point.

File: persons/personsGraph.json (hunk 3 above)
Comment (wsalesky) [reply persons/personsGraph.json:1]: This is for all the persons records, will need to be periodically generated.

File: persons/record.html (hunk 4 above)
Comment (wsalesky) [reply persons/record.html]: See my comments.

Comment (nathangibson) [reply persons/record.html]: So should I just discard the file from the old app? Or only revert the changes in it that you marked?

Comment (wsalesky) [reply persons/record.html]: Discard the file from the old app.

**Comment by nathangibson** [comment]
OK, restored persons/record.html. If everything else is ok, you can merge.

**Comment by wsalesky** [comment]
Done

**Pull request merged**